wagtail/admin/tests/pages/test_edit_page.py

```diff
@@ -2344,14 +2395,90 @@ def test_new_reply(self):
         self.assertEqual(mail.outbox[2].subject, 'test@email.com has updated comments on "I\'ve been edited! (simple page)"')
         self.assertIn('  New replies to: "A test comment"\n   - "a new reply"', mail.outbox[2].body)
 
-    def test_delete_comment(self):
+        # Check audit log
+        log_entry = PageLogEntry.objects.get(action='wagtail.comments.create_reply')
+        self.assertEqual(log_entry.page, self.child_page.page_ptr)
+        self.assertEqual(log_entry.user, self.user)
+        self.assertEqual(log_entry.revision, self.child_page.get_latest_revision())
+        self.assertEqual(log_entry.data['comment']['id'], comment.id)
+        self.assertEqual(log_entry.data['comment']['contentpath'], comment.contentpath)
+        self.assertEqual(log_entry.data['comment']['text'], comment.text)
+        self.assertNotEqual(log_entry.data['reply']['id'], reply.id)
+        self.assertEqual(log_entry.data['reply']['text'], 'a new reply')
+
+    def test_edit_reply(self):
         comment = Comment.objects.create(
             page=self.child_page,
+            user=self.non_subscriber,
+            text="A test comment",
+            contentpath="title",
+        )
+
+        reply = CommentReply.objects.create(
+            comment=comment,
             user=self.user,
+            text='an old reply'
+        )
+
+        post_data = {
+            'title': "I've been edited!",
+            'content': "Some content",
+            'slug': 'hello-world',
+            'comments-TOTAL_FORMS': '1',
+            'comments-INITIAL_FORMS': '1',
+            'comments-MIN_NUM_FORMS': '0',
+            'comments-MAX_NUM_FORMS': '',
+            'comments-0-DELETE': '',
+            'comments-0-resolved': '',
+            'comments-0-id': str(comment.id),
+            'comments-0-contentpath': 'title',
+            'comments-0-text': 'A test comment',
+            'comments-0-position': '',
+            'comments-0-replies-TOTAL_FORMS': '1',
+            'comments-0-replies-INITIAL_FORMS': '1',
+            'comments-0-replies-MIN_NUM_FORMS': '0',
+            'comments-0-replies-MAX_NUM_FORMS': '',
+            'comments-0-replies-0-id': str(reply.id),
+            'comments-0-replies-0-text': 'an edited reply',
+        }
+
+        with enable_comment_panel(SimplePage):
```

Comment:
Do you need `enable_comment_panel` now we don't require the comments data in the formset? Could we just have the comments panel inherited via `settings_panels` as standard?